kubernetes/template/DeploymentTemplates/script.sh

```diff
@@ -431,6 +443,10 @@ echo $METADATA > metadata.json
 ENDPOINT_GRAPH_ENDPOINT=`echo $METADATA | jq '.graphEndpoint' | xargs`
 ENDPOINT_GALLERY=`echo $METADATA | jq '.galleryEndpoint' | xargs`
 ENDPOINT_ACTIVE_DIRECTORY_RESOURCEID=`echo $METADATA | jq '.authentication.audiences'[0] | xargs`
+ENDPOINT_PORTAL=`echo $METADATA | jq '.portalEndpoint' | xargs`
+AZURE_ENV="AzureStackCloud"
```

Comment:
Check around line 359, there is a section to define constants. Could you please move these over there?